### ISSUE TITLE
free: `used` miscalculation

### ISSUE THREAD

**Issue by Krysztal112233** [uutils/procps#36]
Here's procps `free`'s outputs 

```
               total        used        free      shared  buff/cache   available
Mem:        32273048     9797636    15312152      200600     7826228    22475412
Swap:              0           0           0
```

Here's uutils `free`'s outputs

```
               total        used        free      shared  buff/cache   available
Mem:        32273048    16985112    15287936      199512     7825372    22451428
Swap:              0           0           0
```

**Comment by Krysztal112233** [comment]
I'll fix this issues in #35 .

**Comment by Krysztal112233** [comment]
Fixed.
